From d64d7a469a8524d2ea8121d10844ff73feef3b55 Mon Sep 17 00:00:00 2001
From: Peter Ross <pross@xvid.org>
Date: Tue, 1 Mar 2022 19:50:19 +1100
Subject: [PATCH] Assume that EDOM exists

Since errno values are not numeric constants on serenity, this won't
work in cpp, assume that it exists.
---
 libavutil/error.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavutil/error.h b/libavutil/error.h
index 0d3269a..a54d80d 100644
--- a/libavutil/error.h
+++ b/libavutil/error.h
@@ -37,7 +37,7 @@
 
 
 /* error handling */
-#if EDOM > 0
+#if 1
 #define AVERROR(e) (-(e))   ///< Returns a negative error code from a POSIX error code, to return from library functions.
 #define AVUNERROR(e) (-(e)) ///< Returns a POSIX error code from a library function error return value.
 #else
-- 
2.36.1

